.github/workflows/test.yaml

```diff
@@ -3,19 +3,30 @@ name: Tests
 on:
   workflow_call:
     inputs:
-        tox-ini-dir:
-          type: string
-          description: Path to tox.ini file
-          default: './'
+      working-directory:
+        type: string
+        description: The working directory for jobs
+        default: './'
+    outputs:
+      docker-lint-outcome:
```

Comment:
Why this outputs?

Reply:
For testing purposes, to be able to check whether a job ran as expected